packages/toolpad-components/package.json

```diff
@@ -31,7 +31,8 @@
   "dependencies": {
     "@mui/material": "^5.10.17",
     "@mui/toolpad-core": "^0.0.30",
-    "@mui/x-data-grid-pro": "^5.17.15"
+    "@mui/x-data-grid-pro": "^5.17.15",
+    "react-markdown": "^8.0.3"
```

Comment:
Not sure about this dependency: https://github.com/mui/material-ui/issues/12290#issuecomment-453930042